WordPress/Classes/ViewRelated/Gutenberg/Layout Picker/LayoutPickerFilterCollectionViewCell.swift

```diff
@@ -0,0 +1,108 @@
+import UIKit
+
+class LayoutPickerFilterCollectionViewCell: UICollectionViewCell {
+
+    static var cellReuseIdentifier: String {
+         return "LayoutPickerFilterCollectionViewCell"
+     }
+
+    static var nib: UINib {
+        return UINib(nibName: "LayoutPickerFilterCollectionViewCell", bundle: Bundle.main)
+    }
+
+    static var font: UIFont {
+        return WPStyleGuide.fontForTextStyle(.subheadline, fontWeight: .semibold)
+    }
```

Comment:
And I'd also go for a constant here.

Reply:
This one I'd like to leave as a computed property as the font will change based on the device's current accessible font size setting.

Reply:
Good point!